test/api_integration/apis/telemetry/telemetry_local.ts

```diff
@@ -46,46 +31,110 @@ export default function ({ getService }: FtrProviderContext) {
+            { root: ossRootTelemetrySchema, plugins: ossPluginsTelemetrySchema },
+            stats
+          );
+        } catch (err) {
+          err.message = `The telemetry schemas in 'src/plugins/telemetry/schema/' are out-of-date, please update it as required: ${err.message}`;
+          throw err;
+        }
+      });
+
+      it('should pass ad-hoc enforced validations', () => {
+        expect(stats.collection).to.be('local');
+        expect(stats.collectionSource).to.be('local');
+        expect(stats.license).to.be(undefined); // OSS cannot get the license
+        expect(stats.stack_stats.kibana.count).to.be.a('number');
+        expect(stats.stack_stats.kibana.indices).to.be.a('number');
+        expect(stats.stack_stats.kibana.os.platforms[0].platform).to.be.a('string');
+        expect(stats.stack_stats.kibana.os.platforms[0].count).to.be(1);
+        expect(stats.stack_stats.kibana.os.platformReleases[0].platformRelease).to.be.a('string');
+        expect(stats.stack_stats.kibana.os.platformReleases[0].count).to.be(1);
+        expect(stats.stack_stats.kibana.plugins.telemetry.opt_in_status).to.be(false);
+        expect(stats.stack_stats.kibana.plugins.telemetry.usage_fetcher).to.be.a('string');
+        expect(stats.stack_stats.kibana.plugins.stack_management).to.be.an('object');
+        expect(stats.stack_stats.kibana.plugins.ui_metric).to.be.an('object');
+        expect(stats.stack_stats.kibana.plugins.ui_counters).to.be.an('object');
+        expect(stats.stack_stats.kibana.plugins.application_usage).to.be.an('object');
+        expect(stats.stack_stats.kibana.plugins.kql.defaultQueryLanguage).to.be.a('string');
+        expect(stats.stack_stats.kibana.plugins.localization).to.be.an('object');
+        expect(stats.stack_stats.kibana.plugins.csp.strict).to.be(true);
+        expect(stats.stack_stats.kibana.plugins.csp.warnLegacyBrowsers).to.be(true);
+        expect(stats.stack_stats.kibana.plugins.csp.rulesChangedFromDefault).to.be(false);
+
+        // Testing stack_stats.data
+        expect(stats.stack_stats.data).to.be.an('object');
+        expect(stats.stack_stats.data).to.be.an('array');
+        expect(stats.stack_stats.data[0]).to.be.an('object');
+        expect(stats.stack_stats.data[0].pattern_name).to.be('filebeat');
+        expect(stats.stack_stats.data[0].shipper).to.be('filebeat');
+        expect(stats.stack_stats.data[0].index_count).to.be(1);
+        expect(stats.stack_stats.data[0].doc_count).to.be(0);
+        expect(stats.stack_stats.data[0].ecs_index_count).to.be(0);
+        expect(stats.stack_stats.data[0].size_in_bytes).to.be.a('number');
+      });
+
+      it('should validate mandatory fields exist', () => {
+        const actual = flatKeys(stats);
+        expect(actual).to.be.an('array');
+        const expected = [
+          'cluster_name',
+          'cluster_stats.cluster_uuid',
+          'cluster_stats.indices.analysis',
+          'cluster_stats.indices.completion',
```

Comment:
Are we stuck with a static list here?

Reply:
This was an existing test ensuring some mandatory fields (I just moved it inside the new `describe` to group it with the other payload-validation tests). I bet this list is outdated, and we may need to revisit it.